src/Ontology/Signer.cpp

```diff
@@ -27,13 +27,18 @@ Proto::SigningOutput Signer::sign(const Proto::SigningInput& input) noexcept {
         } else if (contract == "ONG") {
             auto encoded = OngTxBuilder::build(input);
             output.set_encoded(encoded.data(), encoded.size());
+        } else {
+            // then assume it's oep4 address
+            auto encoded = Oep4TxBuilder::build(input);
+            output.set_encoded(encoded.data(), encoded.size());
         }
     } catch (...) {
     }
     return output;
 }
 
-Signer::Signer(TW::PrivateKey priKey) : privateKey(std::move(priKey)) {
+Signer::Signer(TW::PrivateKey priKey)
+    : privateKey(std::move(priKey)) {

```

Comment:
Can you rename the variable to `privKey` ?

Reply:
Changed